_oasis

```diff
@@ -398,6 +398,20 @@ Executable "bap-objdump"
                   fileutils,
                   re.posix
 
+Executable "bap-byteweight"
+  Path:           src/readbin
+  MainIs:         byteweight_util.ml
+  Install:        true
+  CompiledObject: best
+  BuildDepends:   bap, bap.plugins, cmdliner, fileutils, curl, re.posix
+
+Executable "bap-compare"
```

Comment:
please rename it to `bap-measure-byteweight` or something like this. It is not `bap-compare` it doesn't compare `bap`, but only `byteweight`.